lib/Debugger.ProtocolHandler/RuntimeImpl.h

```diff
@@ -21,32 +21,48 @@ namespace JsDebug
         RuntimeImpl& operator=(const RuntimeImpl&) = delete;
 
         // protocol::Runtime::Backend implementation
-        void evaluate(
```

Comment:
I reordered the implemented methods alphabetically to make adding them easier. The order is unchanged in the implementation file.